.github/workflows/test-env-deploy.yml

```diff
@@ -241,8 +241,9 @@ jobs:
       - name: Install Dependencies
         if: steps.cache-node-modules.outputs.cache-hit != 'true'
         run: NODE_OPTIONS=--max_old_space_size=4096 npm install
+
       - name: Cypress run critical
-        uses: cypress-io/github-action@v4
+        uses: cypress-io/github-action@v5
```

Comment:
Add  - continue-on-error: true . When test fails next steps are skipped and results are not send do testmo